presentation/src/main/java/me/androidbox/presentation/login/viewmodel/LoginViewModel.kt

```diff
@@ -18,7 +18,7 @@ class LoginViewModel @Inject constructor(
     private val loginUseCase: LoginUseCase
 ) : ViewModel() {
 
-    private val loginScreenMutableState = MutableStateFlow(AuthenticationScreenState<Login>("", "", false, "",null))
+    private val loginScreenMutableState = MutableStateFlow(AuthenticationScreenState<LoginUser>("", "", false, "",null))
```

Comment:
I like to initialize the default values on the actual class, or in this case, you could simply make it so initial value is null

Reply:
@MKiperszmid Sets the default values on the state class